app/http.php

```diff
@@ -646,6 +648,8 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
 
             $activeRuntimes->del($runtimeName);
 
+            $message = \mb_substr($message, -1 * 1024 * 1024); // Limit to 1MB
```

Comment:
shouldn't this be -1 * 1000 * 1000 ?

Reply:
That would be 1 megabit, not 1 megabyte, as per my understanding. Its typical to use bytes when talking about sizes. bits are usually only used when talking about transfer speed.

Reply:
Setting to 1000